ui/config/environment.js

```diff
@@ -31,7 +31,7 @@ module.exports = function (environment) {
 
   if (environment === 'development') {
     ENV['ember-cli-mirage'] = {
-      enabled: true,
+      enabled: ENV.isDevMirageEnabled,
```

Comment:
Does this require any documentation or do most users run the app directly against the conduit API anyways?

Reply:
Not a blocker, just curious by the way! :-)

Reply:
No documentation required. We originally ran it in passthru mode anyway by default, so this is the same thing, but just fully disables mirage. Most if not all users should be running the app directly against the conduit API

Reply:
Neat and thank you for the outline!